CHANGELOG

```diff
@@ -1,4 +1,30 @@
 # Changelog
+## [0.2.5] - 2020-02-19
+### Breaking changes
+ - Deprecate positional arguments in the following methods:
+   - "libnmstate.apply()"
+   - "libnmstate.commit()"
+   - "libnmstate.rollback()"
+   - "libmstate.show()"
+
+### New Features
+ - Report missing NM plugin during validation.
+ - Support OVS Link Aggregation (LAG).
+
+### Bug Fixes
+ - Fix "AtributeError" when editing an existing OVS interface.
+ - Do not remove slaves implicitly when removing the master interface.
+ - Fix team interface port support.
+ - Prevent virtual interface removal after DHCP or IPV6-RA timeout.
+
+### Known Issues
+
+In nmstate 0.2.5 there are some known issues:
+ - Cannot activate a connection against unmanaged network interfaces.
+ - Cannot reapply when bridge profile previously does not contain 802-3-ethernet section.
+ - Cannot set bond option to "arp_validate=active".
```

Comment:
This should mention which bond. Is it OVS Bond or Linux Bond?

Reply:
Is Linux Bond, I am specifying it. Thanks!